app/models/miq_schedule.rb

```diff
@@ -4,6 +4,7 @@ class MiqSchedule < ApplicationRecord
   include Filters
 
   include YamlImportExportMixin
+  # TODO: towhat is a virtual alias?
```

Comment:
deprecate_attribute shows up an alias and virtual attribute when I add instrumentation to virtual attributes... This TODO is really "do we have proper tests to know if rails 7.1 did or did not also break these...?"